recipe/meta.yaml

```
@@ -39,7 +39,10 @@ requirements:
     - {{ stdlib("c") }}
     - cmake
     - make  # [unix]
-    - pkg-config
+    # In OpenMPI, the compiler wrappers are binaries and the wrappers in build
+    # can use host libraries by adding OPAL_PREFIX and in mpich, compiler
+    # wrappers are bash scripts and wrappers in build can't use host libraries.
+    - {{ mpi }}  # [not win and build_platform != target_platform and mpi == "openmpi"]    - pkg-config
```

Looks like this snuck in

```suggestion
    - {{ mpi }}  # [not win and build_platform != target_platform and mpi == "openmpi"]
```